<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ravi-Gayu" failures="1" tests="3" name="ListenerTest" time="17.685" errors="0" timestamp="2024-07-31T10:04:07 IST">
  <testcase classname="com.testng.listeners.OrangeHRM" name="testLogo" time="1.435"/>
  <testcase classname="com.testng.listeners.OrangeHRM" name="testURL" time="0.031">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at com.testng.listeners.OrangeHRM.testURL(OrangeHRM.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testURL -->
  <testcase classname="com.testng.listeners.OrangeHRM" name="testTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testTitle -->
</testsuite> <!-- ListenerTest -->
